[SECURITY][ResponseOps][Alerting][8.19] Alert Deletion Docs (#6952)
elastic/security-docs: docs/detections/alerts-ui-manage.asciidoc

NOTE: Refer to <<timelines-ui>> for information on creating Timelines and Timeline
templates. For information on how to add Timeline templates to rules, refer to <<rules-ui-create>>.

[discrete]
[[clean-up-alerts-sec]]
=== Clean up alerts

> Manage the size of alert indices in your space by clearing out alerts that are older or infrequently accessed. You can do this by {kibana-ref}/run-alert-clean-up-task.html/[running an alert cleanup task], which deletes alerts according to the criteria that you define.

Author: ```suggestion
Manage the size of alert indices in your space by clearing out alerts that are older or infrequently accessed. You can do this by {kibana-ref}/run-alert-clean-up-task.html[running an alert cleanup task], which deletes alerts according to the criteria that you define.
```

> Manage the size of alert indices in your space by clearing out alerts that are older or infrequently accessed. You can do this by {kibana-ref}/view-alerts.html#clean-up-alerts[running an alert cleanup task], which deletes alerts according to the criteria that you define.

Reviewer: Alt suggestion, totally optional. Otherwise LGTM! 
```suggestion
To manage the size of alert indices in your space, run an {kibana-ref}/view-alerts.html#clean-up-alerts[alert cleanup task], which deletes alerts according to the criteria that you define.
```
